--- a/config/extension.toml
+++ b/config/extension.toml
@@ -39,28 +39,3 @@
 # Default values for settings required by this extension.
 [settings]
 app.viewport.defaults.fillViewport = false # Disable the viewport "Fill" option by default.
-
-# Default values for settings applied to the first spectator stream of the primary viewport LDRColor AOV.
-[[settings.exts."omni.kit.livestream.aov"."Render.OmniverseKit.HydraTextures.omni_kit_widget_viewport_ViewportTexture_0.LdrColor".spectatorStream]]
-allowDynamicResize = true   # Allow dynamic resizing of the video stream? All AOV streams enable dynamic resizing by default because the viewport can be resized by the user.
-streamType = "webrtc"       # Type of streaming implementation to create. webrtc=standard WebRTC protocol, native=proprietary. Must be defined for the stream to be created.
-#signalPort = 49101         # TCP port used for negotiating a connection. Must be unique for each stream, if not specified an attempt will be made to use an unnocupied port.
-#streamPort = 48000         # UDP port used to transport streaming media. Must be unique for each stream, if not specified an attempt will be made to use an unnocupied port.
-
-# Additional spectator streams for the primary viewport LDRColor AOV can be specified using multiple array entries.
-#[[settings.exts."omni.kit.livestream.aov"."Render.OmniverseKit.HydraTextures.omni_kit_widget_viewport_ViewportTexture_0.LdrColor".spectatorStream]]
-#streamType = "webrtc"
-#signalPort = 49102
-#streamPort = 48002
-
-# Default values for settings applied to the first spectator stream of the second viewport LDRColor AOV.
-[[settings.exts."omni.kit.livestream.aov"."Render.OmniverseKit.HydraTextures.omni_kit_widget_viewport_ViewportTexture_1.LdrColor".spectatorStream]]
-streamType = "webrtc"
-#signalPort = 49103
-#streamPort = 48004
-
-# Additional spectator streams for the second viewport LDRColor AOV can be specified using multiple array entries.
-#[[settings.exts."omni.kit.livestream.aov"."Render.OmniverseKit.HydraTextures.omni_kit_widget_viewport_ViewportTexture_1.LdrColor".spectatorStream]]
-#streamType = "webrtc"
-#signalPort = 49104
-#streamPort = 48006
